### DISCUSSION TITLE
paging-with-aspnet-mvc/

### DISCUSSION THREAD

**Discussion by martijnboland** [martijnboland/blog#22] [Blog comments]
# paging-with-aspnet-mvc/

https://blogs.taiga.nl/martijn/posts/paging-with-aspnet-mvc/

**Comment by disqus-comments-migrator** [comment]
<em>Troy Goode commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at augustus 28 2008, 02:40</em>

---

<p>Nicely done, Martijn. I'll go ahead and update my blog entry to use IPagedList&lt;t&gt; (implementing IList&lt;t&gt;) rather than the current IPagedList and add a link to this article. Thanks! =)</p>

**Comment by disqus-comments-migrator** [comment]
<em>martijn commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at augustus 28 2008, 03:15</em>

---

<p>Thanks :)</p>

**Comment by disqus-comments-migrator** [comment]
<em>Miguel commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at september 14 2008, 07:33</em>

---

<p>Hi,<br><br>I am just using your code. It is great ... I made a few changes:<br><br>1. Added a new parameter to the Pager: IDictionary&lt;string, object=""&gt; htmlAttributes;<br><br>    This way HTML attributes can be used on the pager. Applied to which tag? See (2)<br><br>2. Wrapped all tag in an unordered list (ul) where each page button is a list item ... In my opinion this is better; I used:<br><br>      TagBuilder pager = new TagBuilder("ul");<br>      pager.MergeAttributes&lt;string, object=""&gt;(attributes);<br><br>3. Added a "box" before the buttons saying "Page {PageNumber} of {PageCount};<br><br>4. Hide next and previous page if PageCount &lt;= 1<br><br>In my page I use some text like "Page ... of ...", or a title for each button like "Page 1", "Next Page", "Previous Page" ...<br><br>What would be the best way to add localization to this items?<br><br>Thanks,<br><br>Miguel</p>
